fuseimg/datasets/cmmd.py

```diff
@@ -76,20 +79,18 @@ def dynamic_pipeline():
         return dynamic_pipeline
 
     @staticmethod
-    def create_dataset_partition(phase : str,
-                                data_dir: str,
-                                data_source : pd.DataFrame,
-                                cache_dir : str = None,
-                                restart_cache : bool = True,
-                                specific_ids : List = []) :
+    def dataset(phase : str,
+                data_dir: str,
+                data_source : pd.DataFrame,
+                cache_dir : str = None,
+                restart_cache : bool = True) :
         """
         Creates Fuse Dataset single object (either for training, validation and test or user defined set)
         :param phase:                       parition name (training / validation / test)
         :param data_dir:                    dataset root path
         :param data_source                  csv file containing all samples file paths and ground truth
```

Comment:
Can you get sample_ids and not csv file?

Reply:
I won't know how to locate the image like this